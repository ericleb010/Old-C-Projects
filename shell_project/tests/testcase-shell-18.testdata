9five=alpha
!six=beta
elev(n=gamma
echo ${9five}
echo ${_six}
echo ${eleven}
