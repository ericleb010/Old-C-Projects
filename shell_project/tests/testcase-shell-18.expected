ERROR: 9five is not a valid variable name. Must not lead with characters other than alphabetics and '_' characters.
ERROR: #six is not a valid variable name. Must not lead with characters other than alphabetics and '_' characters.
ERROR: elev(n is not a valid variable name. Must not contain characters other than alphanumerics and '_' characters.
ERROR: Cannot retrieve 9five, it has not been declared.
ERROR: Cannot retrieve _six, it has not been declared.
ERROR: Cannot retrieve eleven, it has not been declared.

------------------------------------
Welcome to the Assignment Shell!
At any time, you can call "help" to read the documentation.

	>>> 	>>> 	>>> 	>>> 	>>> 	>>> 	>>> 
